src/confluent_kafka/src/Consumer.c

```diff
@@ -97,8 +103,8 @@ static PyObject *Consumer_subscribe (Handle *self, PyObject *args,
 					 PyObject *kwargs) {
 
 	rd_kafka_topic_partition_list_t *topics;
-	static char *kws[] = { "topics", "on_assign", "on_revoke", NULL };
-	PyObject *tlist, *on_assign = NULL, *on_revoke = NULL;
+	static char *kws[] = { "topics", "on_assign", "on_revoke", "on_lost", NULL };
```

Comment:
Why do we want a specific callback for on_lost in the Python client, we're not doing that anywhere else, are we?

Reply:
the only client I know of without a specific callback for lost partitions is the one you made :-).